bpf/bpf_netdev.c

```diff
@@ -393,9 +396,12 @@ static inline int handle_ipv4(struct __sk_buff *skb)
 	return TC_ACT_OK;
 }
 
+#define CB_SRC_IDENTITY 0
```

Comment:
Would be nice to organise this next to the `bpf/lib/common.h` _CB*_ declarations just so it's easier to trace them through the code.

Reply:
I actually want to move away from that as the single declaration doesn't make it clear what it is used where. I'd rather list it above the tail call to make it super clear which cb fields are being used.

Reply:
Ah yep, cool. We can move towards that.